kettle/troubleshoot.md

```diff
@@ -0,0 +1,39 @@
+## Troubleshooting Kettle data collection issue
+Once in a while we have run into situation where there's no data in the Bigquery metrics dashboard in Velodrome ([Issue](https://github.com/kubernetes/test-infra/issues/8703))
+We suspect this to be due to Kettle pod running out of disk space. While we debug the rootcause, here's how you can kickstart the Kettle pod to get it going again.
+
+* CD into k8s/test-infra/kettle
+* make get-cluster-credentials 
+* kubectl get pods // copy Pod Name from the output
+* kubectl logs <Pod-name from above> 
+
+You will see a failure message indicating when the build started failing
+
+```
+==== 2018-07-06 08:19:05 PDT ========================================
+PULLED 174
+ACK irrelevant 172
+EXTEND-ACK  2
+gs://kubernetes-jenkins/pr-logs/pull/kubeflow_kubeflow/1136/kubeflow-presubmit/2385 True True 2018-07-06 07:51:49 PDT FAILED
+gs://kubernetes-jenkins/logs/ci-cri-containerd-e2e-ubuntu-gce/5742 True True 2018-07-06 07:44:17 PDT FAILURE
+ACK "finished.json" 2
+Downloading JUnit artifacts.
+```
+
+Alternatively, navigating to [Gubernator BigQuery page](https://bigquery.cloud.google.com/table/k8s-gubernator:build.all?pli=1&tab=details) (click on “all” on the left and “Details”) you can see a table showing last date/time the metrics were collected.
+
+* kubectl delete pod/<Pod-name from above>
+
+Wait for a couple of minutes for the pod to restart.
+
+* kubectl get pods // should show a new pod name
+```
+NAME                      READY     STATUS    RESTARTS   AGE
+kettle-5df45c4dcb-pf5l2   1/1       Running   2          2h
+```
+* kubectl logs -f <Pod-name from above> // should indicate the pod starting up and collecting data from various GCS buckets. 
```

Comment:
similarly can use JSONPath to script this and not need to have copy-paste manual steps